app/models/ability.rb

```diff
@@ -30,6 +30,7 @@ def initialize(user)
       can [:manage], Client do |client|
         client.provider && user.provider_id.casecmp(client.provider.consortium_id)
       end
+      cannot [:transfer], Client
```

Comment:
I still don't understand this. Is `transfer` included in `can [:manage]`. I thought it is not.

Reply:
@mfenner, As far I can see `manage` includes any action on the Object. Therefore Transfer is https://github.com/ryanb/cancan/wiki/Defining-Abilities#the-can-method

if one remove that then the test for checking ability will fail https://github.com/datacite/lupo/blob/5800f730a1724b05cbca58692011cfc00c5552ab/app/models/ability.rb#L33

Reply:
Thanks for clarification, that makes sense then,